Libbna/CUSTOM-CMS
# #239: fixed card height

**Testing Steps:**

- Go to the home page and check the height of the cards.
- check the height of the cards on smaller devices as well.
- Go to the article page and check the height of the on smaller devices as well.
- Check for the hover effects as well.
- Refer to Screenshot.

**Homepage:**
![image](https://user-images.githubusercontent.com/81369580/150349734-f9805c78-3ba4-4ac3-bad4-a810a271130d.png)

**Article Page:**
![image](https://user-images.githubusercontent.com/81369580/150349809-5e6e6264-5bec-4320-9903-be9556eff2f7.png)
